includes/wikia/models/NavigationModel.class.php

```diff
@@ -1,16 +1,17 @@
 <?php
+
 /**
  * Model for Wiki Navigation
  *
  * @author Jakub "Student" Olek
  */
-
 class NavigationModel extends WikiaModel {
 	const WIKIA_GLOBAL_VARIABLE = 'wgOasisGlobalNavigation';
 	const WIKI_LOCAL_MESSAGE = 'Wiki-navigation';
-	const CACHE_TTL = 10800; // 3 hours
+	// 3 hours
+	const CACHE_TTL = 10800;
 
-	const version = '1';
+	const CACHE_VERSION = '1';
 	const ORIGINAL = 'original';
 	const PARENT_INDEX = 'parentIndex';
 	const CHILDREN = 'children';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "children" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)